<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tsm_cd</class>
 <widget class="QDialog" name="tsm_cd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>238</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>238</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sql watcher - connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/prefix1/icons/d.png</normaloff>:/new/prefix1/icons/d.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="textLabel2">
         <property name="text">
          <string>D&amp;river:</string>
         </property>
         <property name="buddy">
          <cstring>comboDriver</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel3">
         <property name="text">
          <string>Database Name:</string>
         </property>
         <property name="buddy">
          <cstring>editDatabase</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4">
         <property name="text">
          <string>&amp;Username:</string>
         </property>
         <property name="buddy">
          <cstring>editUsername</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4_2">
         <property name="text">
          <string>&amp;Password:</string>
         </property>
         <property name="buddy">
          <cstring>editPassword</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5">
         <property name="text">
          <string>&amp;Hostname:</string>
         </property>
         <property name="buddy">
          <cstring>editHostname</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2">
         <property name="text">
          <string>P&amp;ort:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxPort</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboDriver"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editDatabase">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editUsername">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="editPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/new/prefix1/icons/eye.png</normaloff>:/new/prefix1/icons/eye.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="editHostname"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPort">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="specialValueText">
          <string>Default</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbSaveConnection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save connection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Connect</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
